src/open_inwoner/js/components/cases/index.js

```diff
@@ -0,0 +1,16 @@
+class DisableSubmitButton {
+  constructor(form) {
+    this.form = form
+    this.form.addEventListener('submit', this.disableButton.bind(this))
+  }
+
+  disableButton() {
+    const submitButton = this.form.querySelector('button[type="submit"]')
+    submitButton.setAttribute('disabled', 'true')
```

Comment:
in the Cases styles you could add something similar to this:

`.button[type="submit"]:disabled {
    background-color: var(--color-primary-darker);
    color: var(--color-gray)
  }`

Reply:
Thanks for the hint. I added something similar using the same pattern we use for other disabled buttons in the project.